docs/core/toc.yml

```diff
@@ -363,7 +367,9 @@
   - name: Port WPF projects
     href: porting/wpf.md
   - name: Port C++/CLI projects
-    href: porting/cpp-cli.md
+    href: porting/cpp-cli.md    
+- name: Dependency management
```

Comment:
should this be combined into the section below?

Reply:
We would have to name the section something other than "Dependency loading" if we put "Dependency management" in it.

Reply:
I combined them by making the non-leaf node cover both as "Dependency management and loading"